frontend/src/router/index.js

```diff
@@ -308,13 +313,13 @@ export const createRouter = (app) => {
 
     // Secured pages shouldn't be available to unauthenticated users
     if (to.meta.requiresAuth && !authenticated) {
-      store.commit('alert/setErrorAlert', `You are not authorized to access ${to.path}. Please login first.`)
+      //store.commit('alert/setErrorAlert', `You are not authorized to access ${to.path}. Please login first.`)
```

Comment:
Is this missing the replacement pinia setting of the alert? alertStore.setErrorAlert(). Can remove commented line.

Reply:
Yes.